packages/tailwindcss-language-server/tests/completions/completions.test.js

```diff
@@ -586,8 +586,12 @@ withFixture('v4/basic', (c) => {
 
     expect(resolved).toEqual({
       ...item,
-      detail:
-        'font-size: var(--text-sm); line-height: var(--tw-leading, var(--text-sm--line-height)); letter-spacing: undefined; font-weight: undefined;',
+      detail: [
+        'font-size: 0.875rem /* 8.75px */;',
+        'line-height: ≈ 1.4286;',
```

Comment:
I wonder if this symbol here is going to be confusing or not (because it's not in a comment).

But I do like it though haha

Reply:
I went back-and-forth on this one for this very reason. I think keeping it is OK. @reinink what do you think?

Reply:
Hmm, I kind of think we can just leave the symbol out and just show the approximate value. Alternatively, could we just show the value here that's in the theme like `calc(1.25 / 0.875)`? To me that's the most readable version.